src/main/java/de/geolykt/enchantments_plus/PermissionHandler.java

```diff
@@ -0,0 +1,22 @@
+package de.geolykt.enchantments_plus;
+
+import org.bukkit.command.CommandSender;
+
+public class PermissionHandler {
+
+
+    public static boolean hasPermission(EnchantPlayer player, PermissionTypes permissionType) {
+        return PermissionHandler.hasPermission((CommandSender)player, permissionType);
+    }
+    public static boolean hasPermission(CommandSender player, PermissionTypes permissionType) {
+        String[] permissions =
+            permissionType.getPermissionNames();
+
+        for (String permissionString : permissions) {
+            if (player.hasPermission(permissionString)) {
+                return true;
+            }
+        }
+        return false;
+            }
```

Comment:
Another malindentation
```suggestion
    }
```